docs/custom_datasources_index.rst

```diff
@@ -9,4 +9,12 @@ insights.specs.datasources.cloud_init
 .. automodule:: insights.specs.datasources.cloud_init
     :members: cloud_cfg, LocalSpecs
     :show-inheritance:
-    :undoc-members:
\ No newline at end of file
+    :undoc-members:
+
+insights.specs.datasources.sap
+-------------------------------------
```

Comment:
@xiangce please fix the length of this underline

Reply:
Fixed in the last commit.

Reply:
@bfahr - Would you please review this PR again, I updated it as per #3119. Thanks.